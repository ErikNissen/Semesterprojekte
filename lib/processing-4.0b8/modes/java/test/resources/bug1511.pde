version https://git-lfs.github.com/spec/v1
oid sha256:b464195529f045def3c9d03f2931e409ab58d7eeab71322d30c9190489774ccc
size 47
