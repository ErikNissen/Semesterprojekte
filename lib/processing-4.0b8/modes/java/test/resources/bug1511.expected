version https://git-lfs.github.com/spec/v1
oid sha256:a40f9748e36e8f26f2b59c0e3fe777ccfe32f90e7ec354a6f149d6ef54ccddc2
size 747
